4-projects/README.md

```diff
@@ -0,0 +1,115 @@
+# 4-projects
```

Comment:
Lets add some docs here about how this 4-project differs from example foundation 4-projects

Reply:
I made an update that focuses on the Bank of the Anthos, but I was struggling with how to call out 4-projects in the example foundation without creating confusion.  I'm going to add Manjari to my fork to continue to make updates since I'll be off and on this week.